packages/manager/modules/pci/src/projects/project/kubernetes/index.js

```diff
@@ -0,0 +1,15 @@
+import angular from 'angular';
+
+import kubernetes from '@ovh-ux/manager-kubernetes';
```

Comment:
```suggestion
import '@ovh-ux/manager-kubernetes';
```

Reply:
Nop, it's in the same module

Reply:
Due to the fact that you are importing a peerDependency, I encourage you to import the module as we already done across others modules.

Reply:
You are right ;)